src/views/Files.vue

```diff
@@ -55,7 +55,9 @@ export default {
     ...mapGetters(["selectedCount", "isListing", "isEditor", "isFiles", "isActiveDialog", "shares", "direction", "isSearch"]),
     ...mapState(["req", "user", "reload", "multiple", "loading", "path"]),
     isPreview() {
-      return !this.loading && !this.isListing && !this.isEditor;
+      let preview = !this.loading && !this.isListing && !this.isEditor;
```

Comment:
Are you sure this is the best way to check it? Looks a bit dangerous.. What if there was another option except loading, isListing, isEditor, or preview ? We would have to remember to add it to this condition...

Reply:
change that and use state instead.